data/root/etc/inst_setup

```diff
@@ -55,7 +55,7 @@ unset SSH_FAILED
 stty sane 2>/dev/null
 
 # get hostname & hostips
-hostip_from_wicked >/tmp/host_ips 2>/tmp/host_name
+hostip_from_wicked /tmp/host_ips /tmp/host_name
```

Comment:
Why can't I extract two things from a single wicked run?

host_name is either a fqdn or the ip if no dns is available. I think it's named 'fqhn' sometimes. What's so bad about it?